ScheduleFlow.py

```diff
@@ -525,13 +548,22 @@ def create_gaps_list(self, reserved_jobs, min_ts):
         for event in order_reservations:
             free_nodes = self.system.get_total_nodes() - nodes
             if event[0] > ts and ts != -1 and free_nodes > 0:
```

Comment:
```python
550:   if event[0] > ts ...
```
```python
673:      ts = max(event[0], min_ts)
```
only gaps that start from min_ts are investigated. 
However, gaps that start before can still hold a job starting from ts (all intersecting jobs need to be looked at)

Function needs to be updated to return only possible gaps between min_ts and end of reservation window (for backfill a further optimization is possible, the end window can be the end of the current ending job)

Reply:
This is the reason for the result difference in utilization and response time for the 2018-11-26 testcase.

<pre>
Old <strong>22140</strong> 2.427 0.000 147.628 0.007 ScheduleFlow.py:575(fit_job_in_schedule)
New <strong>6306</strong> 0.188 0.000 10.252 0.002 ScheduleFlow.py:555(fit_job_in_schedule)
</pre>

fit in schedule is called 3x less with the new algorithm because more jobs are fitted at each try